src/views/token-swap/components/selected-quote/selected-quote.view.jsx

```diff
@@ -22,22 +22,6 @@ function SelectedQuote({
 }) {
   const classes = useSelectedQuoteStyles();
 
-  function getRewardAmountInTokens() {
-    return getFixedTokenAmount(
```

Comment:
Token swap use different rewards. They are not related to this issue

Reply:
Thx! Reverted!